demo-app/src/lib/uqi-sources.ts

```diff
@@ -1,8 +1,23 @@
 import createKustoUqiClient from '@/lib/kusto-uqi-client'
 import createMysqlUqiClient from '@/lib/mysql-uqi-client'
+import createSqliteUqiClient from '@/lib/sqlite-uqi-client'
 import createTrinoUqiClient from '@/lib/trino-uqi-client'
+import { type UqiClient } from '@open-truss/open-truss'
 
-const sources = {
+type SourceConfig = any // TODO: make a type that can represent any source config
```

Comment:
Ran out of time getting the types for this right in my time-boxed session so I added. TODO for later.